phenotrex/structure/records.py

```diff
@@ -1,15 +1,16 @@
 #
 # Created by Lukas Lüftinger on 2/5/19.
 #
-from typing import List
+from typing import List, Optional
 from dataclasses import dataclass
 
-"""Data structures containing Genotype and Phenotype information."""
-
 
 @dataclass
 class GenotypeRecord:
-    """ TODO add docstring """
+    """
+    Structure containing the genomic features to which learning is applied, for each sample
+    referenced by `identifier`.
+    """
```

Comment:
```suggestion
    """
    Genomic features of a sample referenced by `identifier`.
    """
```
I'd omit learning here, as the actual feature space depends on all GenotypeRecords in the training set, right?